src/powerwrapper_papi/nrmpower_papi.c

```diff
@@ -311,15 +321,15 @@ int main(int argc, char **argv)
 				                                // logical ID
 
 				if (is_NUMA_event(event)) {
-					nrm_send_progress(
-					        ctxt, watts_value,
-					        nrm_numa_scopes[numa_id]);
+					scope = nrm_numa_scopes[numa_id];
 				} else {
-					nrm_send_progress(
-					        ctxt, watts_value,
-					        nrm_cpu_scopes[numa_id]);
+					scope = nrm_cpu_scopes[numa_id];
 				}
-				verbose("%-45s%4.2f J (Average Power %.2fW)\n",
+				nrm_client_send_event(
+				        client, nrm_time_fromns(elapsed_time),
```

Comment:
time_fromns(elapsed_time) is wrong here, send_event needs a timestamp and this is a time difference. `after_time` has the right value.